stopcovid/dialog/models/events.py

```diff
@@ -400,7 +403,7 @@ def apply_to(self, dialog_state: DialogState):
 
 class SchedulingDrillRequestedSchema(DialogEventSchema):
     @post_load
-    def make_next_drill_requested(self, data, **kwargs):
+    def to_dataclass(self, data, **kwargs):
```

Comment:
I thought it was odd that all of these methods had different names when they do the same transformation so I made them consistent. Thoughts?